src/gluonts/block/scaler.py

```diff
@@ -48,7 +52,7 @@ def compute_scale(self, F, data: Tensor, observed_indicator: Tensor):
             API in MXNet.
 
         data
-            tensor of shape (N, T, C) containing the data to be scaled
+            tensor of shape (N, T, C) or (N, C, T) containing the data to be scaled.
```

Comment:
With the suggested change above, this could become just “tensor containing the data to be scaled”